files/N+1/deployment.tex

```diff
@@ -0,0 +1,8 @@
+\section{Deployment model}
+The deployment model is meant to illustrate where the different software packages are deployed, as shown on figure \ref{deploymentmodel}.
+
+The \glsi{nativescript} codebase is compiled into 2 different file extensions i.e. \verb+.jar+ for \textit{Android} and \verb+.app+ for \textit{iOS}. The compiled files are then deployed directly to the respectively device or uploaded to each platforms individual \gls{appstore}.
```

Comment:
'the respectively device' -> 'their respective devices' 
"platform" -> "platform's"